crux_core/src/core/mod.rs

```diff
@@ -119,7 +119,7 @@ where
 // ANCHOR_END: resolve_sig
     {
         let resolve_result = request.resolve(result);
-        debug_assert!(resolve_result.is_ok());
+        debug_assert!(resolve_result.is_ok(), "{:?}", resolve_result);
```

Comment:
I'm tempted to say "just delete that" 🤔 Can't fully remember what its value was, it probably predates the exposing of the error handling